graspy/inference/dists.py

```diff
@@ -0,0 +1,17 @@
+import numpy as np
+from scipy.spatial.distance import cdist
+
+
+def euclidean(x):
+    """Default euclidean distance function calculation"""
+    return cdist(x, x, metric="euclidean")
```

Comment:
supposedly this is faster https://scikit-learn.org/stable/modules/generated/sklearn.metrics.pairwise_distances.html

I haven't actually tested for speed though. but I doubt this is a bottleneck?